src/TeachABit.Front/src/components/auth/form/AuhtForm.tsx

```diff
@@ -39,28 +41,47 @@ export default function AuthButton() {
             <Button
                 variant="contained"
                 startIcon={<LoginIcon />}
-                sx={{ width: "80%", margin: "20px" }}
+                sx={{ width: "80%", margin: "20px", backgroundColor: "#922728", fontFamily: 'Poppins, Arial, sans-serif' }}
```

Comment:
Za ovo bi bilo dobro napraviti globalnu klasu tipa .primaryButton tako da na više mjesta možemo koristiti isti stil buttona bez da se traži u kodu šta je točno na tom buttonu.